test/ZendeskApi.Client.IntegrationTests/Factories/ZendeskClientFactory.cs

```diff
@@ -1,6 +1,7 @@
 using Microsoft.Extensions.Options;
 using ZendeskApi.Client.IntegrationTests.Settings;
 using ZendeskApi.Client.Options;
+#pragma warning disable 618
```

Comment:
Why is this included?

Reply:
Because the old ZendeskApiClient is marked as Obsolete and produces a warning which we don't want as we'll remove it eventually. We still want to keep the tests.